src/picongpu/CMakeLists.txt

```diff
@@ -142,18 +142,35 @@ set(LIBS ${LIBS} ${mallocMC_LIBRARIES})
 
 find_package(ZLIB REQUIRED)
 include_directories(SYSTEM ${ZLIB_INCLUDE_DIRS})
-set(LIBS ${LIBS} ${ZLIB_LIBRARIES})
+set(LIBS ${LIBS} ZLIB::ZLIB)
```

Comment:
note: [this target](https://github.com/Kitware/CMake/blob/v3.3.0/Modules/FindZLIB.cmake) does already exist in CMake 3.3+